LocalMultiplayerAgent/MultiplayerServerManager.cs

```diff
@@ -55,12 +55,14 @@ public async Task CreateAndStartContainerWaitForExit(SessionHostsStartInfo start
 
             ISessionHostRunner sessionHostRunner =
                 _sessionHostRunnerFactory.CreateSessionHostRunner(startParameters.SessionHostType, _vmConfiguration, _logger);
+
             // RetrieveResources does a docker pull
-            // if we're running Linux containers on Windows we do not want that, since we might 
-            // have built the image locally but tagged with a fake registry name (e.g. myacr.io/mygame:0.1)
-            // docker pull will try to connect to myacr.io and fail
-            // if this image is not locally built, docker create (executed next) will do a docker pull first
-            if (Globals.GameServerEnvironment == GameServerEnvironment.Windows)
+            // If we're running Linux containers on Windows, we might want to pull image from ACR first to save the image on local.
+            // In this case, you need to simply change the value to true for forcePullFromAcrOnLinuxContainersOnWindows in MultiplayerSettings.json
+            // so if this image is not locally built, docker create will do a docker pull first
```

Comment:
docker create will not do a docker pull (that's why it has been failing)

Reply:
gotcha, thank you for correction!